src/asm/rgbasm.5

```diff
@@ -218,7 +218,10 @@ There are a number of numeric formats.
 .It Gameboy graphics Ta \` Ta 0123
 .El
 .Pp
-Underscores are also accepted in numbers, except at the beginning of one.
+Underscores are also accepted in numbers, except at the beginning of one. This can be useful for grouping digits, like
```

Comment:
Sentence-ending periods *must* be at the end of a line.
```suggestion
Underscores are also accepted in numbers, except at the beginning of one.
This can be useful for grouping digits, like
```